install_scripts/install_lago.sh

```diff
@@ -63,10 +80,12 @@ EOF
 
 post_install_conf_for_lago() {
     echo "Configuring permissions"
+    local user_home
     if [[ "$user" != "root" ]]; then
+        user_home=$(eval echo "~$user")
         usermod -a -G lago "$user"
         usermod -a -G qemu "$user"
-        chmod g+x "/home/$user"
+        chmod g+x "$user_home/$user"
```

Comment:
$user_home already provides you the home dir - why do you need the /$user afterwards?